### PR TITLE
Adding tabs & normalising styles

### PR DESCRIPTION
**Pull request by scoutdb**
Tabs added to UI
Normalised styles added to make styles more consistant across browsers.

### CODE REVIEW

**Review by beanpuppy — COMMENTED** [review]
Suggestion; Can you add semicolons to the end of statements? Unless you've memorised all the rules of [ASI](https://262.ecma-international.org/9.0/#sec-rules-of-automatic-semicolon-insertion) it could lead to undefined behaviour.

Side note - The linter/prettier in the base GCX repo picks this up and fixes it on a pre-commit hook, so maybe we should add it here too.

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+/**
+* `getOffsetWidth` returns the sum of collection widths to the left of a given index 
+*/
+function getOffsetWidth(selectedIndex:number, collection: HTMLCollection){
+  let offset = 0 
+  for (let i = 0; i <= selectedIndex -1; i++) {
+    const  w = collection[i].getBoundingClientRect().width
+    w != undefined ? offset += w : 0
```
[the remark this answers is not in the document]
Comment (beanpuppy) [reply src/components/Tabs/Tabs.tsx]: Also prefer `!==` over `!=`.

File: src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+ */
+
+
+const Tabs = (props: TabsProps) => {
+  const TabsRef = React.useRef<HTMLDivElement>(null);
+  
+  // set defaults 
+  const [selected, setSelected] = useState<{
+    value: string | undefined,
+    index: number,
```
Comment (beanpuppy) [inline src/components/Tabs/Tabs.tsx]: Question; Is the `index` key used anywhere? I can't see any references to it.

**Comment by scoutdb** [comment]
> Suggestion; Can you add semicolons to the end of statements? Unless you've memorised all the rules of [ASI](https://262.ecma-international.org/9.0/#sec-rules-of-automatic-semicolon-insertion) it could lead to undefined behaviour.
> 
Good catch - had prettier turned off.

> Side note - The linter/prettier in the base GCX repo picks this up and fixes it on a pre-commit hook, so maybe we should add it here too.
^ think this is a great suggestion

**Review by beanpuppy — CHANGES_REQUESTED** [review]

**Review by beanpuppy — APPROVED** [review]

**Pull request merged**